Add ormar support (#46)
awtkns/fastapi-crudrouter: docs/en/docs/backends/ormar.md

When generating routes, the `OrmarCRUDRouter` will automatically tie into your database

> using your [ormar](https://collerek.github.io/ormar/) models. To use it, you must pass a
> [pydantic](https://pydantic-docs.helpmanual.io/) schema and your `ormar` database model to
> it.

Reviewer: ```suggestion
using your [ormar](https://collerek.github.io/ormar/) models. To use it, simply pass your ormar database model as the schema.
```
